### ISSUE TITLE
Is it possible to configure the size of the preview pane?

### ISSUE THREAD

**Issue by Integralist** [zellij-org/zellij#1778]
I love opening up a preview pane, but it's just a bit too small for my use cases, and so being able to configure the dimensions would be super helpful.

Thanks! Loving Zellij, thank you for sharing ❤️

**Comment by raphCode** [comment]
Hey, thanks for contributing :)

What do you mean by the preview pane / What do you do to open it?

**Comment by Integralist** [comment]
AHHH!!!!! Thanks @imsnif 🙂 that worked great.

**Comment by SimonMonecke** [comment]
Sorry, but I have to ask again because I am not completely happy with the answer.
The problem is that it is very annoying to manually enlarge the window again and again.
I open the floating pane with `zellij run`.
So it would be nice if there would be a parameter for `zellij run` or there would be a default value in the configuration.

Or is my workflow wrong? Whenever I want to commit, for example, I run `zellij run --floating -- gitui`

**Comment by imsnif** [comment]
Your workflow is not wrong @SimonMonecke - and I definitely want to allow the size and location of floating panes to be configured (both from the CLI with `zellij run` and from the layouts), but right now it's not possible.

What I would suggest for your flow is to keep the floating pane open and then toggle its visibility with `ctrl+p` + `w`. Then you only have to enlarge it the first time.

I'm reopening this issue and renaming it to reflect this.

**Comment by sebastiancarlos** [comment]
I agree that this would be a very nice feature. A configuration could re-use the `x, y, width, height` values of. floating panes on layouts.

**Comment by YuFei29** [comment]
> Your workflow is not wrong @SimonMonecke - and I definitely want to allow the size and location of floating panes to be configured (both from the CLI with `zellij run` and from the layouts), but right now it's not possible.
> 
> What I would suggest for your flow is to keep the floating pane open and then toggle its visibility with `ctrl+p` + `w`. Then you only have to enlarge it the first time.
> 
> I'm reopening this issue and renaming it to reflect this.

I have a similar requirement. I wonder what prevents us to implement it

**Comment by aldanor** [comment]
> I agree that this would be a very nice feature. A configuration could re-use the x, y, width, height values of floating panes on layouts.

Usually, even huge monitors are ultra-wide and not ultra-high, so the main problem is height which is too precious of a space to chop it into thirds for floating windows.

Even if there was a single config option like `default_floating_height = "85%"`, this could already be a humongous win.

**Comment by aadi58002** [comment]
Are there any updates regarding this feature request? I would also like this feature very much.

(Thanks for the great work and effort. I love using zellij and the pleasant experience it provides for it user)

**Comment by janbiasi** [comment]
Just came across this issue having the same problem; not being able to configure floating pane sizes for custom keybindings is a big bummer when transitioning form tmux (I'm used to running `lazygit` in a floating pane). 

I was quite surprised that this issue's still open, have there been any alternative ways to achieve this?

**Comment by smsossah** [comment]
Yes this would definitely be a nice feature add to have the ability to resize and position the panes as desired by default prior to opening zellij (in my case I wanted each floating window to appear at the top right of my screen since I am constantly moving it to that location when running unit tests)
